OpenApi.yml

```diff
@@ -716,6 +722,13 @@ components:
       required:
       - collateral
       type: object
+    pkg.GetReportsPeginPegoutRequest:
+      properties:
+        endDate:
```

Comment:
a GET request is supposed to not have body... You should use the query params for this